src/components/BorderlessInput.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react'
+import './BorderlessInput.css'
+
+interface Props {
+    id: string
+    placeholder: string
+    value: string
+    onChange: (value: string) => void
+}
+
+interface State {
+    active: boolean
+}
+
+class component extends React.Component<Props> {
+    state: State = {
+        active: false
+    }
+
+    onFocusInput = () => {
+        console.log(`onClickInput`)
+        this.setState({
+            active: true
+        })
+    }
+
+    onBlurInput = () => {
+        console.log(`onBlur`)
+        this.setState({
+            active: false
+        })
+    }
+
+    onChangeInput = (event: React.ChangeEvent<HTMLInputElement>) => {
+        const inputValue = event.target.value
+        this.props.onChange(inputValue)
```

Comment:
any reason to add the const?  Could just set directly?

Reply:
Set what?
Did you mean this: `this.props.onChange(event.target.value)`

If so, I will change it. For some reason I thought it made it more clear by adding a name for what the `event.target.value` is but on second look it seems obvious since we have the React generic  `HTMLInputElement` so it must be an input value.

Reply:
yeah, that's what I meant